cluster/examples/vm-slirp.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: kubevirt.io/v1alpha1
+kind: VirtualMachine
+metadata:
+  creationTimestamp: null
+  labels:
+    special: vm-slirp
+  name: vm-slirp
+spec:
+  domain:
+    devices:
+      disks:
+      - disk:
+          bus: virtio
+        name: registrydisk
+        volumeName: registryvolume
+      - disk:
+          bus: virtio
+        name: cloudinitdisk
+        volumeName: cloudinitvolume
+      interfaces:
+      - name: testSlirp
+        slirp:
+          Ports:
+          - name: http
+            podPort: 80
```

Comment:
Guided by the question: What does a user want to do? Expose a port.

In that sense we should primarily ask for the port to expose, and provide an optional mechanism to override the podPort.

Thus:

```yaml
name: http
port: 80
protocol: TCP
podPort: 8080  # If it should bind to != 80 on the pod.
```